mne/io/brainvision/brainvision.py

```diff
@@ -331,19 +331,24 @@ def _read_vmrk_events(fname, event_id=None, trig_shift_by_type=None):
             if cur_shift is not None:
                 trigger += cur_shift
             else:
+                # The trigger has been deliberately shifted to None. Do not
+                # add this to "dropped" so we do not warn about something
+                # that was done deliberately. Just continue with next item.
                 trigger = None
+                continue
```

Comment:
This `continue` is needed so that when a user specifies to shift a trigger to `None` (i.e., deleting it) there is no warning raised that this trigger will be dropped. With the `continue`, a warning is only raised if a trigger is dropped unintentionally.

See also the contingent changes in the tests that expected warnings that are no longer raised now.